obspy/io/ascii/core.py

```diff
@@ -44,7 +44,7 @@
 from obspy.core.util import AttribDict, loadtxt
 
 
-HEADER = "TIMESERIES %s_%s_%s_%s_%s, %d samples, %d sps, %.26s, %s, %s, %s\n"
+HEADER = "TIMESERIES %s_%s_%s_%s_%s, %d samples, %s sps, %.26s, %s, %s, %s\n"
 
```

Comment:
```
In [1]: "%lg" % (12.123)
Out[1]: '12.123'

In [2]: "%lg" % (12.0)
Out[2]: '12'
```

so essentially writing "20 sps" instead of "20.0 sps" as in the given examples of IRIS -also  thats what IRIS does in its mseed2ascii scripts - and we should try to stay close to that as they defined that format and who knows which import program assume an integer an this position ... anyway up to you - just my 2 cents